include/h2o/socket/uv-binding.h

```diff
@@ -23,6 +23,7 @@
 #define h2o__uv_binding_h
 
 #include <uv.h>
+#include <sys/time.h>
```

Comment:
Is this an unrelated change?

Reply:
It is, but it's intentional, my system gets warnings because `gettimeofday` below isn't defined. I'm happy to submit the fix as a separate PR, if you prefer.